doc/src/user_guide.rst

```diff
@@ -4,181 +4,16 @@
 User Guide
 **********
 
-Getting Started
-===============
+This will provide infomation on :ref:`running-pyfr` and how calcualtion
```

Comment:
Remove 'calcualtion' ?

Reply:
In fact, I would remove this whole bit of text: "This will provide infomation on Running PyFR and how calcualtion the Configuration File (.ini) is setup. For information on how to install PyFR see Installation."